Lib/mutatorMath/objects/mutator.py

```diff
@@ -133,7 +133,7 @@ def getAxisNames(self):
         s = {}
         for l, x in self.items():
             s.update(dict.fromkeys([k for k, v in l], None))
-        return s.keys()
+        return sorted(list(s.keys()))
```

Comment:
And where do you get the impression that "consumers expect a specific sorting" for `getAxisNames`?